pyproject.toml

```
@@ -29,6 +30,9 @@ build-dir = "build/{wheel_tag}"
 # Build stable ABI wheels for CPython 3.12+
 wheel.py-api = "cp312"
 
+cmake.verbose = true
+logging.level = "INFO"
```

```suggestion
logging.level = "DEBUG"
```

Sorry, a bit more info would be helpful.

---

No worries, I committed this change just now.

---

did the more detailed log message offer any useful info?